homeassistant/components/sensor/geizhals.py

```diff
@@ -128,21 +128,21 @@ def update(self):
                            timeout=1)
         soup = bs4.BeautifulSoup(request.text, 'html.parser')
 
-        # parse name
-        raw = soup.find_all('span', attrs={'itemprop': 'name'})
-        self.device_name = raw[1].string
+        # Parse name
+        raw = soup.find('h1', attrs={'class': 'gh-headline'})
```

Comment:
This should be extracted from Home Assistant and into a separate Python library. We will not be able to accept any changes to this platform until this is fixed.

Reply:
Thanks for your comments @MartinHjelmare and @balloob ! I will create a separate library for that.